Makefile

```diff
@@ -45,7 +47,12 @@ require-cython:
 	@[ -n "$(PYTHON_WITH_CYTHON)" ] || { \
 	    echo "NOTE: missing Cython - please use this command to install it: $(PYTHON) -m pip install Cython"; false; }
 
-wheel_manylinux: wheel_manylinux64 wheel_manylinux32
+qemu-user-static:
+	if [ -n "$(WITH_QEMU)" ] ; then \
+		docker run --rm --privileged multiarch/qemu-user-static --reset -p yes; \
```

Comment:
Hmm, wouldn't running `yes` here potentially produce a lot of (log) output somewhere? Would something like `bash -c "while true; do sleep 10; done"` also work? And how does this container get killed after use?

Reply:
This container runs a one-shot script which register qemu-static interpreter and exits immediatly after that.
`yes` is not the bash command but a parameter of `qemu-binfmt-conf.sh` which tells the script to load the interpreter in memory.